bundle/src/main/java/com/adobe/acs/commons/widgets/MultiFieldPanelFunctions.java

```diff
@@ -57,7 +57,7 @@ private MultiFieldPanelFunctions() {
     public static List<Map<String, String>> getMultiFieldPanelValues(Resource resource, String name) {
         ValueMap map = resource.adaptTo(ValueMap.class);
```

Comment:
Great point, thanks for the input.

On Sun, Sep 13, 2015, 05:51 justinedelson <notifications@github.com> wrote:

> In
> bundle/src/main/java/com/adobe/acs/commons/widgets/MultiFieldPanelFunctions.java
> <https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/528#discussion_r39347349>
> :
>
> > @@ -57,7 +57,7 @@ private MultiFieldPanelFunctions() {
> >      public static List<Map<String, String>> getMultiFieldPanelValues(Resource resource, String name) {
> >          ValueMap map = resource.adaptTo(ValueMap.class);
>
> just for future reference (no reason to change this now), since we're now
> requiring AEM 6.0, a slightly better solution would be to replace this line
> with
>
> ValueMap map = resource.getValueMap();
>
> That will guarantee that map is a non-null object.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/528/files#r39347349>
> .
>